astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -216,6 +225,43 @@ def get_roll0(cls):
         # API, so it's better for it to be accessable from Galactocentric
         return _ROLL0
 
+    @classmethod
+    def get_latest(cls, *args, **kwargs):
+        """This classmethod serves as an alternate initializer for
+        `~astropy.coordinates.Galacotcentric` that assumes more modern (and
+        constantly updated) parameters for the solar position and motion in a
+        Galactocentric context. As such, this classmethod acts exactly like the
+        initializer, but with different default frame attribute choices: That
+        is, you can explicitly pass in frame attributes to overwrite these
+        defaults, or pass in data to initialize this frame.
+
+        The current latest parameter choices come from:
+        * The default distance to the Galactic Center is 8.122 kpc from the
+          GRAVITY collaboration:
+          https://ui.adsabs.harvard.edu/abs/2018A%26A...615L..15G/abstract
+        * The default height of the Sun above the Galactic midplane is taken to
+          be 20.8 pc, as measured by Bennett & Bovy (2019):
+          https://ui.adsabs.harvard.edu/abs/2019MNRAS.482.1417B/abstract
+        * The default solar velocity relative to the Galactic center is taken
+          from a combination of the GRAVITY collaboration and the Reid &
+          Brunthaler (2004) proper motion of Sgr A*:
+          https://ui.adsabs.harvard.edu/abs/2004ApJ...616..872R/abstract
+          https://ui.adsabs.harvard.edu/abs/2018RNAAS...2..210D/abstract
+
+        See the docstring for the initializer of this
+        `~astropy.coordinates.Galacotcentric` class for information about the
+        frame and its default component (data) names.
+        """
+
+        kwargs['galcen_distance'] = kwargs.pop('galcen_distance', 8.122 * u.kpc)
```

Comment:
Clearer would be
```
kwargs.setdefault('galcen_distance', 8.122*u.kpc)
(etc)
```
(note: PEP8 avoids the spaces around operators for arguments)